update/files/multiFileWriter_test.go

```diff
@@ -66,7 +71,7 @@ func TestNewFile(t *testing.T) {
 func TestWrite(t *testing.T) {
 	t.Parallel()
 
-	exportFolderPath := "./"
+	exportFolderPath := "./test"
```

Comment:
temp dir?

Reply:
exportFolderPath could remain :).

Reply:
No, I mean using temporary directories.

Reply:
that might hide some errors. I would prefer testing with real files.